core/ExecutePipe.hpp

```diff
@@ -124,6 +124,9 @@ namespace olympia
 
         // Used to flush the ALU
         void flushInst_(const FlushManager::FlushingCriteria & criteria);
+
+        // Friend class used in rename testing
+        friend class ExecutePipeTester;
```

Comment:
Seems this class is missing from the PR

Reply:
Defined in [Rename_Test](https://github.com/riscv-software-src/riscv-perf-model/blob/4fd928d4400af49de00ac3f5f8b5c4990bdbb2fd/test/core/rename/Rename_test.cpp#L102)